issue-wanted.cabal

```diff
@@ -101,7 +101,7 @@ library
                      , unordered-containers
                      , warp ^>= 3.2
 
-executable issue-wanted
+executable issue-wanted-exe
```

Comment:
I think this is a minor mistake introduce during conflict resolution

Reply:
Ooops! Thank you.